pkg/virt-controller/watch/migration_test.go

```diff
@@ -1586,6 +1597,124 @@ var _ = Describe("Migration watcher", func() {
 		})
 
 	})
+
+	Context("Migration monitoring", func() {
+		var (
+			vmi           *virtv1.VirtualMachineInstance
+			migration     *virtv1.VirtualMachineInstanceMigration
+			targetPod     *k8sv1.Pod
+			attachmentPod *k8sv1.Pod
+			labels        prometheus.Labels
+		)
+
+		BeforeEach(func() {
+			vmi = newVirtualMachineWithHotplugVolume("testvmi-monitoring-"+rand.String(16), virtv1.Running)
```

Comment:
I think `rand.String(6)` should be more than enough :)

Reply:
ok, changed